hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -322,17 +322,17 @@
   public static final int OZONE_SCM_PIPELINE_OWNER_CONTAINER_COUNT_DEFAULT = 3;
   // Pipeline placement policy:
   // the max number of pipelines can a single datanode be engaged in.
```

Comment:
Can we update the comments to:
Upper limit for how many pipelines a datanode can engage in?

Reply:
Yea I updated.